src/index.html

```diff
@@ -3,12 +3,280 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Dia</title>
+    <title>DIA</title>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,600,700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Poppins:400,600,700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.css">
+    <link rel="icon" href="./images/logo/flower.png">
+    <script type="module" src="scripts/main.js"></script>
   </head>
   <body>
-    <img src="./images/logo.svg" alt="Logo">
-    <h1>Dia</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+    <header class="header"  id="home">
+      <div class="wrapper wrapper--header">
+        <button class="header__button"></button>
+        <div class="header__nav">
+          <a href="" class="header__link"><img src="./images/logo/Logo.svg" alt="logo"></a>
```

Comment:
href не может быть пустым
логотип обычно ведет на главную страницу
```suggestion
          <a href="/" class="header__link"><img src="./images/logo/Logo.svg" alt="logo"></a>
```